ext/calendar/easter.c

```diff
@@ -21,6 +22,10 @@
 #include "sdncal.h"
 #include <time.h>
 
+/**
+ * If `gm` is true this will return the timestamp at midnight on eastern of the given year. If it is false this
```

Comment:
Thanks for this contribution. I found two minor typos in the comment above the `_cal_easter()` function:

`eastern` -> `Easter`
`march the 21` -> `March 21st`

Reply:
Thank you for the fast review! I fixed those typos.